src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -316,8 +318,8 @@ class TableList extends React.Component<TableListProps, TableListState> {
                 value: item
             });
         });
-        Object.keys(columnSelected).map(key => {
-            const item = columnSelected[key];
+        Object.keys(columnList).map(key => {
```

Comment:
Well... I think most of `Object.keys()` are unnecessary and can be replaced with `forEach()` or something else...

Reply:
Ok, will change in next version.